### PR TITLE
Fixing the fonts for autoblocked blog page

### LINKED ISSUE #13: /blog page autoblocked cards
This will be a JS intensive ticket.
This ticket has a dependency on https://github.com/mkclawson/impala/issues/6 .

There are 2 variations of "cards" that will need to pull in data from the indexed metadata sheet. 
This ticket is for the /blog "home' page, where we have horizontal (wide) listings of the blog articles in Desktop, and we will have vertical cards for mobile.

https://www.supernaturalimpala.com/blog

Resources that you can use as a guide:

This is a "blog home" auto block from Merative:
https://github.com/hlxsites/merative/commit/6790248782f8503a91c656d4fdd7c722179fe1b8#diff-fbceb58f32b99b8a5309251f8c49ac8eda82fba58b905c600c92bce1ed2a1c8c

It depends on this function being added to scripts.js:
https://github.com/hlxsites/merative/blob/6790248782f8503a91c656d4fdd7c722179fe1b8/scripts/scripts.js#L108

### PR DESCRIPTION
**Pull request by meejain**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #13 

Test URLs:
- Before: https://main--impala--mkclawson.hlx.live/
- After: https://issue-13-styling-blogpages--impala--mkclawson.hlx.live/blog/rocky-mountain-snow

### CODE REVIEW

**Review by helms-charity — CHANGES_REQUESTED** [review]
You are missing the dot separators in those autoblocked author/date/readtime elements.
Check spacing to the left of those elements against the live page. Also check spacing between the autoblocked elements and the H1.

**Comment by meejain** [comment]
> You are missing the dot separators in those autoblocked author/date/readtime elements. Check spacing to the left of those elements against the live page. Also check spacing between the autoblocked elements and the H1. image

**Comment by meejain** [comment]
@helms-charity  - The Dots are very much visible. Please check - 

https://issue-13-styling-blogpages--impala--mkclawson.hlx.live/blog/rocky-mountain-snow
https://issue-13-styling-blogpages--impala--mkclawson.hlx.live/blog/garaged
https://issue-13-styling-blogpages--impala--mkclawson.hlx.live/blog/bringin-back-baby
https://issue-13-styling-blogpages--impala--mkclawson.hlx.live/blog/pandemic-woes

I am working on the Updated date information - updating your blog files with that information

**Comment by meejain** [comment]
image

**Comment by helms-charity** [comment]
After opening in Incognito, I see the dots now, but the spacing for me is still off. Very weird. 
image

**Comment by helms-charity** [comment]
OK, I see what is going on. If you make the browser get narrower, the container width is disproportionate to the rest of the content, and it moves around on the top. You have a 750px width but then using 60%. Best to make this match whatever the rest of the content below is doing.

image

**Comment by meejain** [comment]
@helms-charity  - Please check now - 

image

image


https://issue-13-styling-blogpages--impala--mkclawson.hlx.live/blog/rocky-mountain-snow
https://issue-13-styling-blogpages--impala--mkclawson.hlx.live/blog/garaged
https://issue-13-styling-blogpages--impala--mkclawson.hlx.live/blog/bringin-back-baby
https://issue-13-styling-blogpages--impala--mkclawson.hlx.live/blog/pandemic-woes

File: styles/styles.css [hunk 1]
```diff
@@ -303,9 +303,17 @@ main > .section > .default-content-wrapper > p {
   margin: auto;
   max-width: 60%;
   text-align: left;
+  margin-top: 5vh;
 }
 
-.blog main > .section > .default-content-wrapper > p {
+.blog main > .section > .default-content-wrapper > p:first-of-type {
+  font-family: 'GFS Didot', serif;
```
Comment (helms-charity) [inline styles/styles.css:310]: Why only the first-of-type? What if Mike authors more than one paragraph?

Comment (meejain) [reply styles/styles.css:310]: Yes that's why I was recommending "updated" information to be in metadata table rather. That way its styling would have been fixed automatically along with Autoblocked items. @helms-charity

Comment (meejain) [reply styles/styles.css:310]: @helms-charity  - When we updated your blog sheet with the updated information, it was coming as a P element - hence we need to update that P - first-of-type . If that helps further.

File: styles/styles.css [hunk 2]
```diff
@@ -303,9 +303,17 @@ main > .section > .default-content-wrapper > p {
   margin: auto;
   max-width: 60%;
   text-align: left;
+  margin-top: 5vh;
```
Comment (helms-charity) [inline styles/styles.css:306]: Why would you do this? This means that the distance between the autoblogblock changes based on browser height. Make it a PX value that matches the live site.

File: styles/styles.css (hunk 1 above)
Comment (helms-charity) [reply styles/styles.css:310]: Remove this section about 14px please, we don't need it. I will address it another way.

File: styles/styles.css [hunk 3]
```diff
@@ -303,9 +303,17 @@ main > .section > .default-content-wrapper > p {
...
 
-.blog main > .section > .default-content-wrapper > p {
+.blog main > .section > .default-content-wrapper > p:first-of-type {
+  font-family: 'GFS Didot', serif;
+  font-style: italic;
+  font-size: 14px;
+  margin-top: 0;
+}
+
+.blog main > .section > .default-content-wrapper > p:not(:first-of-type) {
```
Comment (helms-charity) [inline styles/styles.css:316]: remove the :not(:first-of-type) from here.

File: styles/styles.css (hunk 1 above)
Comment (meejain) [reply styles/styles.css:310]: @helms-charity  - NM, I figured out a way via CSS itself.

File: styles/styles.css (hunk 3 above)
Comment (meejain) [reply styles/styles.css]: NM .. I have fixed that via other way

File: styles/styles.css (hunk 2 above)
Comment (meejain) [reply styles/styles.css]: That was done just to make things adjustable as per the device size but yes - height doesn't matter much. Done with the changes.

**Review by helms-charity — APPROVED** [review]

**Pull request merged**